nerves_toolchain_ctng/patches/crosstool-ng/0003-Turn-off-PCH-on-Darwin-to-fix-build-error.patch

```diff
@@ -0,0 +1,33 @@
+From 00d3feaf2c2beea980792f20018c4e4901a0ca7b Mon Sep 17 00:00:00 2001
+From: Frank Hunleth <fhunleth@troodon-software.com>
+Date: Tue, 7 Dec 2021 09:08:11 -0500
+Subject: [PATCH 3/3] Turn off PCH on Darwin to fix build error
+
+See https://github.com/riscv-software-src/homebrew-riscv/issues/47#issuecomment-772306695
+---
+ packages/gcc/11.2.0/0009-darwin-no-pch.patch | 13 +++++++++++++
+ 1 file changed, 13 insertions(+)
+ create mode 100644 packages/gcc/11.2.0/0009-darwin-no-pch.patch
+
+diff --git a/packages/gcc/11.2.0/0009-darwin-no-pch.patch b/packages/gcc/11.2.0/0009-darwin-no-pch.patch
+new file mode 100644
+index 00000000..16b89945
+--- /dev/null
++++ b/packages/gcc/11.2.0/0009-darwin-no-pch.patch
+@@ -0,0 +1,13 @@
++--- a/gcc/config.host.orig	2021-12-07 07:28:40.000000000 -0500
+++++ b/gcc/config.host	2021-12-07 07:28:47.000000000 -0500
++@@ -93,8 +93,8 @@
++ case ${host} in
++   *-darwin*)
++     # Generic darwin host support.
++-    out_host_hook_obj=host-darwin.o
++-    host_xmake_file="${host_xmake_file} x-darwin"
+++    #out_host_hook_obj=host-darwin.o
```

Comment:
I don't quite understand this path - these commented out lines are the same as above. Is there a reason to include them?

Reply:
The only reason that I commented these out is to make it exactly match what people were saying at https://github.com/riscv-software-src/homebrew-riscv/issues/47#issuecomment-772306695. I agree that normally I'd just delete them.

I personally don't like this change. It seems like there should be a more precise way of turning off PCH and I'm curious why crosstool-ng's PCH disabling code was insufficient. 

Since gcc-11 doesn't work currently for aarch64 on Apple M1, my plan is to revisit in a month or two to see whether anything changes. I'm not quite sure how to debug gcc-11 issues further right now.